source/codegen/metadata/nixnet/custom_proto.mako

```diff
@@ -0,0 +1,41 @@
+message FlexRayStats {
+    uint32 NumSyntaxErrorChA = 1;
+    uint32 NumSyntaxErrorChB = 2;
+    uint32 NumContentErrorChA = 3;
+    uint32 NumContentErrorChB = 4;
+    uint32 NumSlotBoundaryViolationChA = 5;
+    uint32 NumSlotBoundaryViolationChB = 6;
+}
+
+message J1939CommState{
+    uint32 PGN = 1;
+    uint32  SourceAddress = 2;
+    uint32  DestinationAddress = 3;
+    uint32  TransmitError = 4;
+    uint32  ReceiveError = 5;
+    uint32 Reserved1 = 6;
```

Comment:
Don't include `Reserved` fields in the proto.

Reply:
Removed the reserved fields